server/segment_rpc.go

```diff
@@ -549,3 +555,11 @@ func genPayment(sess *BroadcastSession, numTickets int) (string, error) {
 
 	return base64.StdEncoding.EncodeToString(data), nil
 }
+
+func validatePrice(sess *BroadcastSession) bool {
```

Comment:
What do you think about returning an error instead of a boolean here so all the details about price validation can be encapsulated in this function (rather than having `genPayment()` decide on an appropriate error message for an invalid price)?

Reply:
That's actually a good idea and I don't know why I didn't think of that.

Reply:
implemented in https://github.com/livepeer/go-livepeer/pull/992/commits/18648d78c0d532e7e1cc26d8bc1e374a560cee34